interfaces/builtin/pulseaudio.go

```diff
@@ -0,0 +1,47 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/ubuntu-core/snappy/interfaces"
+)
+
+const pulseaudioConnectedPlugAppArmor = `
+/etc/pulse r,
+/etc/pulse/* r,
+/dev/shm/pulse-shm-* mrwlkix,
```

Comment:
This fails with: 

maj 05 16:58:29 zyga-thinkpad-w510 kernel: audit: type=1400 audit(1462460309.256:26418): apparmor="DENIED" operation="open" profile="snap.flight-of-the-amazon-queen.flight-of-the-amazon-queen" name="/dev/shm/pulse-shm-2512407204" pid=8921 comm="scummvm" requested_mask="r" denied_mask="r" fsuid=1000 ouid=1000